### PR TITLE
Remove frontend

### LINKED ISSUE #27: Remove all non-api front-end view/templates

### PR DESCRIPTION
**Pull request by indocomsoft**
Fixes #27

### CODE REVIEW

File: lib/cadet_web/router.ex [hunk 1]
```diff
@@ -1,8 +1,8 @@
 defmodule CadetWeb.Router do
   use CadetWeb, :router
 
-  pipeline :browser do
-    plug(:accepts, ["html"])
+  pipeline :api do
+    plug(:accepts, ["json"])
     plug(:fetch_session)
     plug(:fetch_flash)
     plug(:protect_from_forgery)
```
Comment (evansb) [inline lib/cadet_web/router.ex]: I don't think you need CSRF protection since cookies is not used

File: lib/cadet_web/router.ex [hunk 2]
```diff
@@ -1,8 +1,8 @@
 defmodule CadetWeb.Router do
   use CadetWeb, :router
 
-  pipeline :browser do
-    plug(:accepts, ["html"])
+  pipeline :api do
+    plug(:accepts, ["json"])
     plug(:fetch_session)
     plug(:fetch_flash)
```
Comment (evansb) [inline lib/cadet_web/router.ex]: This shouldn't be needed anymore

**Comment by indocomsoft** [comment]
Done.

**Comment by indocomsoft** [comment]
@evansb Done :)

**Comment by evansb** [comment]
@indocomsoft can you make a separate PR to set code coverage back to 98-100 percent? remove all dead code, and #26

**Comment by indocomsoft** [comment]
Will do. #33 is one attempt to.

**Pull request merged**